.bit/responses/2.5-Week 2 Step 5.md

```diff
@@ -7,157 +7,99 @@ step: 5
+As with our other function, we need to be able to use `node-fetch`. Outside of the async function, declare the variable `fetch` (like we did in our other function):
+```js
+var fetch = require('node-fetch');
+```
 
-> [Master the JavaScript Interview: What is a Promise?](https://medium.com/better-programming/is-javascript-synchronous-or-asynchronous-what-the-hell-is-a-promise-7aa9dd8f3bfb)
-</br>
-</details>
-<br>
 <details>
-<summary>❓ What is the URL?</summary>
+<summary>❗ Calling the GIPHY API</summary>
 </br>
 
-Notice that the URL is just the uriBase with the params we specified earlier appended on.
-
-For now, fill in the `method`  and `body`.  
-
+Inside the async function, remove most of the code so that it looks like this:
 ```js
-async function analyzeImage(img){
-    
-    const subscriptionKey = '<YOUR SUBSCRIPTION KEY>';
-    const uriBase = '<YOUR ENDPOINT>' + '/face/v1.0/detect';
-
-    let params = new URLSearchParams({
-        'returnFaceId': 'true',
-        'returnFaceAttributes': 'emotion'
-    })
-
-    
-    //COMPLETE THE CODE
-    let resp = await fetch(uriBase + '?' + params.toString(), {
-        method: '<METHOD>',  //WHAT TYPE OF REQUEST?
-        body: '<BODY>',  //WHAT ARE WE SENDING TO THE API?
-        headers: {
-            '<HEADER NAME>': '<HEADER VALUE>'  //do this in the next section
-        }
-    })
-
-    let data = await resp.json();
-    
-    return data; 
+module.exports = async function (context, req) {
+    context.log('JavaScript HTTP trigger function processed a request.');
+
+    context.res = {
+        // status: 200, /* Defaults to 200 */
+        body: //LEAVE THIS BLANK
+    };
 }
 ```
-</details>
-<br>
-<details>
-<summary>❓How do I specify Request Headers?</summary>
-</br>
-
-Go back to the Face API documentation [here](https://westus.dev.cognitive.microsoft.com/docs/services/563879b61984550e40cbbe8d/operations/563879b61984550f30395236), and find the **Request headers** section. 
-
-There are two headers that you need. I've provided the format below. Enter in the two header names and their two corresponding values. 
-
-FYI: The `Content-Type`  header should be set to`'application/octet-stream'`.  This specifies a binary file.
 
+Now, we need to store the dominant emotion (sent in the **body** of the `GET` request) in a variable called `emotion`:
 ```js
-    //COMPLETE THE CODE
-    let resp = await fetch(uriBase + '?' + params.toString(), {
-        method: '<METHOD>',  //WHAT TYPE OF REQUEST?
-        body: '<BODY>',  //WHAT ARE WE SENDING TO THE API?
-      
-      	//ADD YOUR TWO HEADERS HERE
-        headers: {
-            '<HEADER NAME>': '<HEADER VALUE>'
-        }
-    })
+const emotion = (req.query.//WHAT GOES HERE || (req.body && req.body.//WHAT GOES HERE));
 ```
 
-</details>
-<br>
-<details>
-<summary>❓How do I actually analyze the image?</summary>
-</br>
+We also need to store the phone number sent in the **body** of the `GET` request:

```

Comment:
Students won't need to text it anywhere (the gif will just be returned in the function output)